packages/effect/src/Array.ts

```diff
@@ -1106,6 +1106,42 @@ export const findLast: {
   }
 )
 
+/**
+ * Counts all the element of the given iterable that pass the given predicate
+ *
+ * **Example**
+ *
+ * ```ts
```

Comment:
If you could add this to the changelog too thanks :)

Reply:
sure. I just did it!